test/reference/objects/Buildings/Controls/OBC/CDL/Continuous/Sources/Constant.json

```diff
@@ -0,0 +1,393 @@
+                                {
+                                  "x": 82,
+                                  "y": -70
+                                }
+                              ],
+                              "color": {
+                                "r": 192,
+                                "g": 192,
+                                "b": 192
+                              }
+                            }
+                          },
+                          {
+                            "name": "Polygon",
+                            "attribute": {
+                              "points": [
+                                {
+                                  "x": 90,
+                                  "y": -70
+                                },
+                                {
+                                  "x": 68,
+                                  "y": -62
+                                },
+                                {
+                                  "x": 68,
+                                  "y": -78
+                                },
+                                {
+                                  "x": 90,
+                                  "y": -70
+                                }
+                              ],
+                              "lineColor": {
+                                "r": 192,
+                                "g": 192,
+                                "b": 192
+                              },
+                              "fillColor": {
+                                "r": 192,
+                                "g": 192,
+                                "b": 192
+                              },
+                              "fillPattern": "FillPattern.Solid"
+                            }
+                          },
+                          {
+                            "name": "Line",
+                            "attribute": {
+                              "points": [
+                                {
+                                  "x": -80,
+                                  "y": 0
+                                },
+                                {
+                                  "x": 80,
+                                  "y": 0
+                                }
+                              ]
+                            }
+                          },
+                          {
+                            "name": "Text",
+                            "attribute": {
+                              "extent": [
+                                {
+                                  "x": -150,
+                                  "y": -150
+                                },
+                                {
+                                  "x": 150,
+                                  "y": -110
+                                }
+                              ],
+                              "textString": ",textString=",
```

Comment:
should it be a separate issue?

Reply:
Yes. created it here: https://github.com/lbl-srg/modelica-json/issues/213